lib/database/game.php

```diff
@@ -906,25 +906,26 @@ function getTotalUniquePlayers($gameID, $requestedBy)
  * Gets a game's high scorers or latest masters.
  *
  * @param int $gameID game ID to get high score information for
- * @param int $offset query offset value
- * @param int $count query number of returned rows
  * @param string $requestedBy user requesting the information
- * @param int $type The type of data to return.
- *          0 - High Scores
- *          1 - Latest Masters
  *
  * @return array of user information to display on the High Scores section of a game page
  */
-function getGameTopAchievers($gameID, $offset, $count, $requestedBy, $type = 0)
+function getGameTopAchievers($gameID, $requestedBy)
 {
     sanitize_sql_inputs($gameID, $offset, $count, $requestedBy);
 
-    $retval = [];
-    $havingQuery = "";
-    $order = "ASC";
-    if ($type == 1) {
-        $havingQuery = "HAVING TotalScore = (SELECT SUM(Points * 2) AS Points FROM Achievements WHERE GameID = $gameID AND Flags = 3)";
-        $order = "DESC";
+    $high_scores = [];
+    $masters = [];
+    $mastery_score = 0;
+
+    $query = "SELECT SUM(Points * 2) AS Points FROM Achievements WHERE GameID = $gameID AND Flags = 3";
+    $dbResult = s_mysql_query($query);
+    SQL_ASSERT($dbResult);
```

Comment:
This call to `SQL_ASSERT` can be omitted - it only will result in an error log entry which wouldn't serve any meaningful purpose if the query failed for whatever reason.
Not relevant to this PR though - we still have a few of those.

Reply:
Just copied from a few lines down.

Reply:
Got it - i see that we use this quite often even though we don't need it anymore.
Can be ignored